providers/implementations/digests/sha3_prov.c

```diff
@@ -547,8 +552,36 @@ static void *keccak_dupctx(void *ctx)
 
 static const OSSL_PARAM known_shake_settable_ctx_params[] = {
     {OSSL_DIGEST_PARAM_XOFLEN, OSSL_PARAM_UNSIGNED_INTEGER, NULL, 0, 0},
+    {OSSL_DIGEST_PARAM_SIZE, OSSL_PARAM_UNSIGNED_INTEGER, NULL, 0, 0},
```

Comment:
This is the settable array, but the settable function doesn't seem to handle this param